test/integration/helpers/PathToWindowsNode

```diff
@@ -1 +1 @@
-C:/Program Files (x86)/Appium/node_modules/appium/bin/appium.js
```

Comment:
does it still make sense to have two identical resources?

Reply:
I've marked them as obsolete, I'll remove them on the next commit

Reply:
removed